### ISSUE TITLE
FormFlow questions

### ISSUE THREAD

**Issue by manu4rhyme** [Microsoft/BotBuilder#688]
I have some UX related questions for FormFlow

Firstly,
In case of enums, we get buttons, when we click on buttons, we see inputs like 1, 4 etc.
Can we instead get **text of the button clicked** instead of their index?

Secondly, 
When some field is pre-filled, it shows it like, 
**What is your email address ? (Current Choice : abc@microsoft.com)**
Can we instead show something like 
**(Current Choice : abc@microsoft.com) What is your email address ?**
or basically change how pre-filled fields are shown

**Comment by chrimc62** [comment]
Cn you describe why you are looking at button input?  It should automatically bind to the underlying c# enum you are using for a choice field.  Whether they do a button or type a response you should end up with the same result.

Please read the documentation on FormFlow http://docs.botframework.com/en-us/csharp/builder/sdkreference/forms.html It describes how you can override templates on your class or per field to control how things are displayed.

**Comment by manu4rhyme** [comment]
If we have two buttons say,
1) Yes
2) No
When i click on Yes, i see a number 1 being sent to the Bot.
It would be nice if there is an option to show the text, Yes (or no text at all) instead, being sent to the Bot.
Its basically a UX thing, and seeing 1 being sent to the Bot simply doesn't look as good as seeing Yes 
being sent to the Bot. 
If the user himself enters 1, then showing 1 is fine for me.

Regarding the documentation for the second question, I saw the TemplateUsage.CurrentChoice and FormConfiguration.CurrentChoice
I've already added "ok" to my list of FormConfiguration.CurrentChoice, and it seems to be working fine.

I also want to change the string that displays the current choice. Do i have to set prompt manually, based on the current form state? Or is there a better way to do it?

Thanks.
